src/context.rs

```diff
@@ -25,38 +24,73 @@ impl Default for Match {
     }
 }
 
-pub struct Context<'a> {
-    schema: &'a Schema,
```

Comment:
`Schema` is no more needed here, since `Schema` uses field name as hash key and `Context` no longer uses field name to add value, but only index in `Router`'s fields array